printrun/projectlayer.py

```diff
@@ -704,8 +746,22 @@ def refresh_display(self, event):
         self.present_first_layer(event)
 
     def update_thickness(self, event):
-        self._set_setting('project_layer', self.thickness.GetValue())
-        self.refresh_display(event)
+        layer = self.thickness.GetValue()
+
+        if ',' in layer:
+            self.statusbar.SetLabel(_("'Layer' contains a comma, please use a point for decimal values."))
+            self.Layout() # Layout() is needed to ellipsize possible overlength status
+            return False
+
+        try:
+            float(layer)
+        except ValueError:
+            self.statusbar.SetLabel(_("Unrecognized number in 'Layer': %s") % layer)
+            return False
+        else:
```

Comment:
1) The `else` clause is unnecessary after a return statement.
2) Should this function return `True` on success? Current it returns nothing if all goes well but returns a boolean if it doesn't